book/specifications/kimchi/Specification.toml

```diff
@@ -10,7 +10,7 @@ template = "template.md"
 [sections]
 # arguments
 permutation = "../../../kimchi/src/circuits/polynomials/permutation.rs"
-lookup = "../../../kimchi/src/circuits/polynomials/lookup.rs"
+lookup = "../../../kimchi/src/circuits/lookup/constraints.rs"
```

Comment:
weirdly CI didn't complaina bout this, need to investigate

Reply:
there was an error, but it didn't stop CI...

https://github.com/o1-labs/proof-systems/runs/5607924312?check_suite_focus=true#step:11:232

<img width="972" alt="Screen Shot 2022-03-25 at 12 54 47 PM" src="https://user-images.githubusercontent.com/1316043/160192110-d56cca46-ecee-4ee0-99ef-f62c563c9fbe.png">

Reply:
pushed a fix (hopefully) as part of this PR